package.json

```diff
@@ -51,7 +51,12 @@
   "module": "lib.esm.js",
   "types": "index.d.ts",
   "dependencies": {
+    "@date-io/date-fns": "^1.3.13",
+    "@date-io/moment": "^1.3.13",
     "@material-ui/core": "^4.10.2",
+    "@material-ui/pickers": "^3.2.10",
```

Comment:
We would also need this as a peer dependency

Reply:
@Hasmik8  we can add the same in `"peerDependencies": {}`